src/Vault.sol

```diff
@@ -0,0 +1,207 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+import "lib/chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol";
+import "lib/yield-utils-v2/contracts/token/IERC20.sol";
+import "lib/openzeppelin-contracts/contracts/access/Ownable.sol";
+
+
+interface IERC20Decimals is IERC20 {
+    function decimals() external view returns (uint);
+}
+
+contract Vault is Ownable {
+
+    ///@dev Vault records collateral deposits of each user
+    mapping (address => uint) public deposits; 
```

Comment:
Nope, he didn't comment on it previously. Will check in with him next week, when he is back

Reply:
I've always used uint256 myself, I felt bad about requiring that because `uint` is shorter than `uint256`, and there is little chance for any confusion.

If we vote on this, I'll vote for `uint256` across the board.

Reply:
lets just stick w uint256

Reply:
Will make the changes

Reply:
Made the changes, [3841c2f](https://github.com/calnix/Collateralized-Vault/pull/4/commits/3841c2f06f6a1bbee2b95305192ee83071cf2ec0)